scripts/private-github-packages.cjs

```
@@ -0,0 +1,485 @@
+
+function parseGhScopes(output) {
+  const scopesLine = output
+    .split(/\r?\n/)
+    .find((line) => /Token scopes:/i.test(line));
+  if (!scopesLine) {
+    return [];
+  }
+  return scopesLine
+    .replace(/^.*Token scopes:\s*/i, '')
+    .split(',')
+    .map((scope) => scope.replace(/[\s'\"]/g, ''))
```

ESLint

[error] Unnecessary escape character: \".

(no-useless-escape)